server/src/test/java/io/crate/integrationtests/JoinIntegrationTest.java

```diff
@@ -1396,4 +1396,49 @@ FROM UNNEST([1, 2, 3]) z
                 "3| NULL"
         );
     }
+
+    /*
+     * https://github.com/crate/crate/issues/13503
+     */
+    @Test
+    @UseHashJoins(1)
+    public void test_nested_joins() {
+        execute("CREATE TABLE doc.j1 (x INT)");
+        execute("CREATE TABLE doc.j2 (x INT)");
+        execute("CREATE TABLE doc.j3 (x INT)");
+
+        execute("insert into doc.j1(x) values (1),(2),(3)");
+        execute("insert into doc.j2(x) values (1),(2),(3)");
+        execute("insert into doc.j3(x) values (1),(2),(3)");
+
+        execute("refresh table doc.j1");
+        execute("refresh table doc.j2");
+        execute("refresh table doc.j3");
```

Comment:
can do in a single call as a comma separated list.

Reply:
👍